src/main/java/blang/distributions/Generators.java

```diff
@@ -28,6 +29,14 @@ public static double halfstudentt(Random random, double nu, double sigma) {
 	  double t = studentt(random, nu);
 	  return Math.abs(t) * sigma;
   }
+  
+  /** */
+  public static double pareto(Random random, double scale, double shape)
+  {
+    double result = new ParetoDistribution(generator(random), scale, shape).sample();
+    return result;
```

Comment:
Simplify to 

```
return new ParetoDistribution(generator(random), scale, shape).sample();
```

I think this pattern in this file is an artifact of some past debugging session. They could actually all be simplified.